samples/apps/bookinfo/route-rule-all-v1.yaml

```diff
@@ -1,13 +1,4 @@
 type: route-rule
-name: productpage-default
```

Comment:
Are there v2 pods available? If yes, then this definitely changes the behavior of the routing.